lib/elastic_searcher.rb

```diff
@@ -59,7 +59,7 @@ def search_definition # rubocop:disable Metrics/MethodLength
       end
 
       # only return gems that are not yanked unless filter is used
-      post_filter { term yanked: false } unless query_str.include? "yanked:true"
+      post_filter { term yanked: false } unless query_str.delete(" ").include? "yanked:true"
```

Comment:
please use `gsub(/[[:space:]]/, "")`

Reply:
Isn't `delete(" ")` much faster for this purpose?
Any reason for `gsub`?

Reply:
spaces can be weird. `[[:space:]]` gets most of them.